themes/default/content/blog/enhanced-search-intuitive-navigation-new-docs-experience/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Enhanced search & Navigation: The new Pulumi Docs experience"
+date: 2023-06-12
+meta_desc:  At Pulumi, we believe in exceptional documentation experiences that help people using Pulumi find what they need quickly and use it successfully.
+meta_image: meta.png
+authors:
+    - susan-evans
+    - anita-trimbur
+    - sean-holung
+tags:
+    - pulumi-releases
+    - features
+    - ux
+---
+
+Engineers spend a lot of time searching documentation for answers. At Pulumi, we believe in exceptional documentation experiences that help people using Pulumi find what they need quickly and use it successfully. Today, we announce an improved Pulumi documentation experience dedicated to all developers.
+<!--more-->
+
+We prioritized search and navigation, so you can easily browse and search for precisely what you want. Enjoy faster and more accurate searches, with lovely user experience touches to guide you to the right place. We also planned for the future, using design patterns that will grow with content and concepts.
+
+![Pulumi docs welcome banner](imgs/banner.png)
+
+Let's showcase a few types of Pulumi documentation:
+
+## Quickstart experience
+
+The [Pulumi quickstart](/docs/quickstart/) experience is for people new to Pulumi to get up and running quickly.
+
+![Pulumi docs quickstart experience](imgs/quickstart.png)
+
+## Cloud get started tutorials
+
+The [Pulumi cloud get started tutorials](/docs/clouds/) are for people who want a more detailed walkthrough when using Pulumi for the first time.
+
+![Pulumi cloud get started tutorials](imgs/get-started.png)
+
+## Concept docs
+
+The [Pulumi concept docs](/docs/concepts/) are where you can dive into Pulumi concepts. Learn how they work together and how to effectively use them to ship infrastructure.
+
+![Pulumi concept docs](imgs/concepts.png)
+
+## Packages
+
+The [package documentation](/docs/registry/) is where to find configuration and API documentation for Pulumi supported clouds, version control systems, databases, infrastructure monitoring, and networking tools. We support 119 providers today, and we add new providers every week!
+
+![Pulumi api packages](imgs/packages.png)
+
+## Search
```

Comment:
Can we move this to the top - partly because it aligns with the title and intro focus on "search and navigation"?